docs/antrea-network-policy.md

```diff
@@ -459,17 +535,30 @@ prohibited` code for ICMP, UDP and SCTP request, or an explicit reject response
 TCP request, instead of timeout. A "Pass" rule, on the other hand, skips this packet
 for further ACNP rule evaluations (all ACNP rules that has lower priority than the
 current "Pass" rule will be skipped, except for the Baseline Tier rules), and delegates
-the decision to developer created namespaced NetworkPolicies. If no NetworkPolicy matches
-this traffic, then the Baseline Tier rules will still be matched against. Note that the
-"Pass" action does not make sense when configured in Baseline Tier ACNP rules, and such
-configurations will be rejected by the admission controller.
+the decision to developer created namespaced NetworkPolicies. If no NetworkPolicy matches this traffic,
+then the Baseline Tier rules will still be matched against. Note that the "Pass" action does not
+make sense when configured in Baseline Tier ACNP rules, and such configurations will be rejected
+by the admission controller. Note: multicast NetworkPolicy does not support "Pass" and "Reject" action.
```

Comment:
"Pass" and "Reject" actions are not supported for rules applied to multicast traffic.